OracleGoldenGate/Dockerfile

```diff
@@ -71,7 +69,8 @@ RUN         if [[ -z "${OGG_VERSION}" || -z "${OGG_EDITION}" || -z "${OGG_TARFIL
 # Additional installation for Oracle GoldenGate Microservices Architecture
 #
             if [[ "${OGG_EDITION}" == "microservices" ]]; then \
-                yum -y --enablerepo *software_collections install openssl rh-nginx18 java-1.8.0-openjdk-headless && \
+                . /etc/os-release && MAJOR_VERSION="${VERSION/.*/}" && \
```

Comment:
This isn't really necessary: we're not going to switch from `oraclelinux:7-slim` as the base image so we know that the `MAJOR_VERSION` will be `7`. I appreciate the attention to detail, but it's just another level of obfuscation that impedes readability (loquacious pun intended).